docs/plan/phase-3/README.md

```
@@ -0,0 +1,22 @@
+# Phase 3: Emails
+
+> [Back to main plan](../README.md)
+
+## Goal
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)